demos/super_resolution_demo/models.lst

```diff
@@ -1,2 +1,3 @@
 # This file can be used with the --list option of the model downloader.
 single-image-super-resolution-????
+text-image-super-resolution-????
```

Comment:
Please comment this for now, as listing a nonexistent model breaks demo tests. We'll have to enable it after the model itself arrives.

Reply:
Ok, waiting when it happens.
But i really have no idea what you are talking about. What do you want to enable? Where should the model arrive?

Reply:
> Ok, waiting when it happens.

Please don't. I would rather merge this now, but for that to happen I need you to comment this line out.

> What do you want to enable?

Poor phrasing on my part, I guess. I meant to say "we'll uncomment it [this line] back after the model config arrives".

> Where should the model arrive?

Into this repository.

Reply:
Oh, ok. Done.